ClassicGuildBankData/Migrations/20190609224636_init.cs

```diff
@@ -252,7 +253,8 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 {
                     SlotId = table.Column<int>(nullable: false),
                     BagId = table.Column<Guid>(nullable: false),
-                    ItemId = table.Column<int>(nullable: true)
+                    ItemId = table.Column<int>(nullable: true),
+                    Notes = table.Column<string>(nullable: true)
```

Comment:
These columns need to be added as a separate migration

Reply:
@athielking I have pushed with `add-migration` command to migrate model changes